packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverActions/useTeachingPopoverActions.ts

```diff
@@ -1,34 +1,25 @@
 import * as React from 'react';
-import { getIntrinsicElementProps, slot } from '@fluentui/react-utilities';
+import { getIntrinsicElementProps } from '@fluentui/react-utilities';
 import type { TeachingPopoverActionsProps, TeachingPopoverActionsState } from './TeachingPopoverActions.types';
 
 /**
- * Create the state required to render TeachingPopoverActions.
- *
- * The returned state can be modified with hooks such as useTeachingPopoverActionsStyles_unstable,
- * before being passed to renderTeachingPopoverActions_unstable.
- *
- * @param props - props from this instance of TeachingPopoverActions
- * @param ref - reference to root HTMLDivElement of TeachingPopoverActions
+ * Returns the props and state required to render the component
+ * @param props - TeachingPopoverActions properties
+ * @param ref - reference to root HTMLElement of TeachingPopoverActions
  */
 export const useTeachingPopoverActions_unstable = (
   props: TeachingPopoverActionsProps,
-  ref: React.Ref<HTMLDivElement>,
+  ref: React.Ref<HTMLElement>,
```

Comment:
Is there a reason why it's `HTMLElement`? Because of that you have to cast below

Reply:
Set back to div element, this was something we inherited from popover but its not needed.